Source/GameView/GameViewModels/GameDataViewModel.swift

```diff
@@ -39,13 +32,18 @@ public class GameDataViewModel {
    
    - Returns: A tuple of the necessary details for the view to display.
    */
-  public func getGameDetails(at index: Int) -> (name: String, gamePrice: String) {
+  public func getGameDetails(at index: Int) -> (gameName: String, gamePrice: String) {
+    
+    let game = model.getGameAt(index: index)
     
-    if let game = gameManager?.gameList[index] {
-      return (game.name, "R\(game.price)")
-    }
+    let gameDetails = (gameName: game.name, gamePrice: "R\(game.price)")
+    
+    return gameDetails
     
-    return ("No Name", "No price")
+  }
+  
+  public func getGameAt(at index: Int) -> Game {
```

Comment:
Same idea as above here, there's no reason to say get game at at. We only really need to say it once prefereably like so:
`getGame(at index: Int)`